static_src/util/validators.js

```diff
@@ -41,7 +41,8 @@ export function validateString() {
 
 export function validateEmail() {
   return function _validateEmail(value, name) {
-    if (!(/^[a-zA-Z0-9.!#$%&amp;'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/).test(value)) {
+    const emailRegex = /^[a-zA-Z0-9.!#$%&amp;'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
We should move this outside the exported function, this will still re-parse the regex every time the function is called. Its not a big deal performance-wise but I think its good practice